naver_land/base_crawler.py

```diff
@@ -0,0 +1,17 @@
+import requests
+
+
+class BaseCrawler:
+    def __init__(self):
+        self.baseURL = "https://m.land.naver.com/complex"
+
+    def setup_header(self):
+        header = {
+            "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36"
+        }
+        return header
+
+    def get_request(self, url: str) -> requests.Response:
+        header = self.setup_header()
```

Comment:
```suggestion
    def __init__(self):
        self.baseURL = "https://m.land.naver.com/complex"
        self.headers = self.setup_header()

    def setup_header(self):
        header = {
            "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36"
        }
        return header

    def get_request(self, url: str) -> requests.Response:
        header = * self.headers *
```

header 세팅이 변하지 않는 값이라면,
매번 새로 함수를 호출하는 것 보다  crawler의 속성값으로 가지고 있어도 될 것 같아용

Reply:
그게 좋을 것 같네요!